spec/acceptance/virtualenv_spec.rb

```diff
@@ -111,5 +103,57 @@ class { 'python' :
       # but probability of this happening is minimal, so it should be acceptable.
       apply_manifest(pp, catch_changes: true)
     end
+    it 'works with editable=>true' do
+      pp = <<-EOS
+      package{ 'git' :
+        ensure => 'present',
+      }
+      -> class { 'python' :
+        version    => 'system',
+        pip        => 'present',
+        virtualenv => 'present',
+      }
+      -> python::virtualenv { 'venv' :
+        ensure     => 'present',
+        systempkgs => false,
+        venv_dir   => '/opt/venv5',
+        owner      => 'root',
+        group      => 'root',
+      }
+      -> python::pip { 'rpyc' :
+        ensure     => '4.1.0',
+        url        => 'git+https://github.com/tomerfiliba/rpyc.git',
+        editable   => true,
+        virtualenv => '/opt/venv5',
+      }
+      EOS
+
+      # Run it twice and test for idempotency
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
+    it 'works with == in pkgname' do
+      pp = <<-EOS
+      'python' :
```

Comment:
I think something got lost here?

```suggestion
      class{'python' :
```

Reply:
Ouch! While I'm add it, would it be preferred if I removed the leading space before all resource names?

Also, I found what's causing the first two tests to fail: using $path in the exec instead of $_path, going to push that along with this correction.

Reply:
Removing the leading space isn't required and I think it would just make the diff harder to read. I think we can keep that as is.